app/controllers/users_controller.rb

```diff
@@ -1,4 +1,26 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
```

Comment:
I am going to crucify you .... make the redirect and flash a one liner.

Reply:
Using it this way gives me an easy way to leverage bootstrap alert classes like here:
https://stackoverflow.com/a/46594462